__tests__/integration/configuration/config-fixture.js

```diff
@@ -20,7 +20,7 @@ const config = {
   consoleLevel: 'silly',
   tokenUid: '',
   gapLimit: null,
-  connectionTimeout: null,
+  connectionTimeout: 30000,
```

Comment:
Added on [10e0b6b](https://github.com/HathorNetwork/hathor-wallet-headless/pull/164/commits/10e0b6bb65dcca9f2acd3ff536aa83457c2b7f95)

Reply:
This is great for people who may be studying this code, so that they don't think providing a `connectionTimeout` parameter is important for common wallet usage.

Docs updated on [988283f](https://github.com/HathorNetwork/hathor-wallet-headless/pull/164/commits/988283fc03c015cdebd66cdd33883afda365edd4)